tools/occlum

```diff
@@ -129,20 +129,43 @@ cmd_init() {
     mkdir -p image
     mkdir -p image/bin
     mkdir -p image/lib
+    mkdir -p image/lib64
     mkdir -p image/root
     mkdir -p image/host
     mkdir -p image/tmp
+    local occlum_glibc_lib=/usr/local/occlum-glibc/lib
+    local cpu_lib=/sys/devices/system/cpu
+    if [ -d "$occlum_glibc_lib" ]; then
+        mkdir -p "image/$occlum_glibc_lib"
+        mkdir -p "image/$cpu_lib"
+    fi
     # add default /etc/hosts
     mkdir -p image/etc
     echo "127.0.0.1   localhost" > image/etc/hosts
 
-    local occlum_gcc_lib=/usr/local/occlum/x86_64-linux-musl/lib
-    cp -t image/lib/ \
+    # add musl
+    local occlum_musl_lib=/usr/local/occlum/x86_64-linux-musl/lib
+    cp -t image/lib \
         /lib/ld-musl-x86_64.so.1 \
-        "$occlum_gcc_lib/libc.so" \
-        "$occlum_gcc_lib/libstdc++.so.6" \
-        "$occlum_gcc_lib/libgcc_s.so.1" \
-        "$occlum_gcc_lib/libgomp.so.1"
+        "$occlum_musl_lib/libc.so" \
+        "$occlum_musl_lib/libstdc++.so.6" \
+        "$occlum_musl_lib/libgcc_s.so.1" \
+        "$occlum_musl_lib/libgomp.so.1"
```

Comment:
Since we demand copying more and more files, maybe we should consider using symbolic links instead.